pkg/authorization/authorizer/scope/converter.go

```diff
@@ -170,12 +171,15 @@ func DescribeScopes(scopes []string) map[string]string {
 type userEvaluator struct{}
 
 func (userEvaluator) Handles(scope string) bool {
-	return strings.HasPrefix(scope, UserIndicator)
```

Comment:
I was expecting something like `return e.Validate(scope) == nil` since that forces the logic into one place.

Reply:
it's the same just makes more semantic sense inside Validate() to check for Handles() rather than in Handles() to call Validate()

Reply:
ok